.env.alfajores

```diff
@@ -23,20 +23,23 @@ CELOSTATS_BANNED_ADDRESSES=""
 CELOSTATS_RESERVED_ADDRESSES=""
 
 ORACLE_DOCKER_IMAGE_REPOSITORY="us-west1-docker.pkg.dev/celo-testnet-production/celo-oracle/celo-oracle"
-ORACLE_DOCKER_IMAGE_TAG="1.0.3"
+ORACLE_DOCKER_IMAGE_TAG="latest"
```

Comment:
Do we want to keep this on latest?

Reply:
Yeah will do, as that's what's currently deployed on testnets. Will update it soon anyways once we do a new release